uniffi/src/lib.rs

```diff
@@ -16,19 +16,25 @@
 //!  * How to ["lower"](ViaFfi::lower) rust values of that type into an appropriate low-level
 //!    FFI value.
 //!  * How to ["lift"](ViaFfi::lift) low-level FFI values back into rust values of that type.
-//!  * How to [write](ViaFfi::write) rust values of that type into a bytebuffer, for cases
+//!  * How to [write](ViaFfi::write) rust values of that type into a buffer, for cases
 //!    where they are part of a compount data structure that is serialized for transfer.
-//!  * How to [read](ViaFfi::read) rust values of that type from bytebuffer, for cases
+//!  * How to [read](ViaFfi::read) rust values of that type from buffer, for cases
```

Comment:
FYI, a lot of minor changes here are around trying to use consistent terminology across the different backends, saying "rust buffer" when we mean a `Vec<u8>` owned by rust.